web/modeltr/request.py

```diff
@@ -9,6 +9,7 @@ class Request(Document):
     type = trString
     state = trString
     machine = trString
+    entity = trString
```

Comment:
Ivo explained as "part of machine", though for screenshots/snapshots.

Reply:
Consider subpart/subentity - TODO

Reply:
subject_id is good name to be used

Reply:
fixed